### ISSUE TITLE
Prototypical Contrastive Learning of Unsupervised Representations 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#10]
# TL;DR
The leading representation learning methods, SimCLR and Moco, should utilize a large number of negative samples and so both require huge batch sizes. However, PCL, which is based on EM algorithm uses cluster centers of samples instead of large number of negative ones , solves this problem, and surpasses Moco and others.
![image](https://user-images.githubusercontent.com/24524018/89389767-c919a000-d740-11ea-8be8-53d4921f4916.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2005.04966

# Submission Dates(yyyy/mm/dd)
2020/05/11

# Authors and institutions
Junnan Li, Pan Zhou, Caiming Xiong, Richard Socher, Steven C.H. Hoi 
Salesforce Research 

# Methods
In E step, clustering and In M step minimizing the loss function. 
![image](https://user-images.githubusercontent.com/24524018/89389794-d33b9e80-d740-11ea-9bd5-57bb766eefb4.png)

They use centers of clusters but still need negative samples.
![image](https://user-images.githubusercontent.com/24524018/89389803-d767bc00-d740-11ea-9294-8612d73ed5c4.png)

# Results
![image](https://user-images.githubusercontent.com/24524018/89389776-cdde5400-d740-11ea-9b41-b446066aeecf.png)

# Comments
